stdlib/src/utils/string_slice.mojo

```diff
@@ -192,6 +192,44 @@ fn _is_valid_utf8(data: UnsafePointer[UInt8], length: Int) -> Bool:
     return _validate_utf8_simd_slice[4, True](ptr, length, iter_len) == 0
 
 
+fn _isnewline_start(
+    ptr: UnsafePointer[UInt8], read_ahead: Int = 1
+) -> (Bool, Int):
+    """Returns if the first item in the pointer is the start of
+    a newline sequence, and its length.
+    """
+    # TODO add line and paragraph separator as stringliteral
+    # once unicode escape secuences are accepted
```

Comment:
```suggestion
    # once Unicode escape sequences are accepted
```

Reply:
damn I've already written that in other PRs :( I'll find all and fix them